rpc/erigon.go

```diff
@@ -274,7 +278,7 @@ func (client *ErigonClient) GetBlock(number int64) (*types.Eth1Block, *types.Get
 				tracePb.From = common.FromHex(trace.Action.Address)
 				tracePb.To = common.FromHex(trace.Action.RefundAddress)
 				tracePb.Value = common.FromHex(trace.Action.Balance)
-			} else if trace.Type == "call" || trace.Type == "delegatecall" {
```

Comment:
**trace.Type** cannot be "delegatecall". The different call types are in **trace.Action.CallType**.